internal/server/storage/drivers/driver_zfs.go

```diff
@@ -567,6 +567,20 @@ func (d *zfs) importPool() (bool, error) {
 		return false, err
 	}
 
+	// We need to explicitly import the keys here so containers can start. This
+	// is always needed because even if the admin has set up auto-import of
+	// keys on the system, because incus manually imports and exports the pools
+	// the keys can get unloaded.
+	//
+	// We could do "zpool import -l" to request the keys during import, but we
+	// also should give warnings to users if they've configured their ZFS
+	// datasets to have encryption setups that are incompatible with how incus
+	// manages its ZFS pools.
+	if err := d.loadKeys(d.config["zfs.pool_name"]); err != nil {
```

Comment:
Our coding style is to avoid the one line define+check syntax, can you split that into two separate statements?